config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml

```diff
@@ -16,10 +16,7 @@ limber_scrna_core_cdna_prep_v2:
     - LRC Bank Seq
     - LRC Bank Spare
   library_types:
-    - Chromium single cell 5 prime GEM-X
-    # - Chromium single cell 3 prime GEM-X
-    # - Chromium single cell BCR GEM-X
-    # - Chromium single cell TCR GEM-X
+    - Chromium single cell 5 prime GEM-X GE v3
```

Comment:
I added 'v3' here because it was expected by the integration suite test. However, I'm not sure if we should include it or not - there's an ongoing conversation about it with Abby by email.

Reply:
Removed. Info from Abby:

> it sounds like the version numbers may be redundant for us? I’m not sure why the current line has version numbers as Sarah told me that the version numbers refer to the overall offerings i.e. Next-GEM chips were one version, GEM-X another. 10x don’t seem to release versions within a chip type.
I think the version no. was captured for records but yes I think based on the above, any version change would be significant enough to require a lot of changes and testing and probably result in a more obvious name change anyway.